src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreTargetGraph.cs

```diff
@@ -17,52 +17,48 @@ public class RestoreTargetGraph : IRestoreTargetGraph
         /// <summary>
         /// Gets the runtime identifier used during the restore operation on this graph
         /// </summary>
-        public string RuntimeIdentifier { get; internal set; }
+        public string RuntimeIdentifier { get; }
 
         /// <summary>
         /// Gets the <see cref="NuGetFramework" /> used during the restore operation on this graph
         /// </summary>
-        public NuGetFramework Framework { get; internal set; }
+        public NuGetFramework Framework { get; }
```

Comment:
I don't see a shipped.txt change here, I'm guessing it's what's failing the build

Reply:
This didn't change the public API

Reply:
Oh duh, internal :D my bad.